pkgs/development/compilers/dotnet/update.sh

```diff
@@ -283,7 +283,7 @@ update() {
     if [ "$patch_specified" == false ] && [ -f "$output" ]; then
         local -a versions
         IFS= readarray -d '' versions < <(
-            nix-instantiate --eval --json -E "with (import $output {
+            nix-instantiate --eval --json -E "with (import \"$output\" {
```

Comment:
This is a big hacky.  I'm not sure if there's a good way to escape a nix string from bash.

Reply:
Worst case you could probably use `builtins.readFile`, but i don't know if that's really needed.

Reply:
I've changed it to use `--argstr`, which should be robust.